src/pinned_memory_manager.cc

```diff
@@ -376,4 +391,16 @@ PinnedMemoryManager::Free(void* ptr)
   return instance_->FreeInternal(ptr);
 }
 
+uint64_t
+PinnedMemoryManager::GetTotalPinnedMemoryByteSize()
+{
+  return pinned_memory_byte_size_;
+}
+
+uint64_t
+PinnedMemoryManager::GetUsedPinnedMemoryByteSize()
+{
+  return used_pinned_memory_byte_size_;
```

Comment:
Are these accesses thread-safe? It seems like the regions where it is updated are protected by `info_mtx_`, which implies some concurrent usage to me.

Reply:
Thank you for noting this. Since `info_mtx_` is an instance variable and cannot be used in static methods, created a specific mutex variable for reading and writing `used_pinned_memory_byte_size_` without modifying the existing `info_mtx_`.